.env

```diff
@@ -2,7 +2,7 @@ PG_DATABASE_URL=postgres://granite:password@localhost:5432/granite_db
 MYSQL_DATABASE_URL=mysql://granite:password@localhost:3306/granite_db
 SQLITE_DATABASE_URL=sqlite3:./granite.db
 CURRENT_ADAPTER=pg
-PG_VERSION=10.4
+PG_VERSION=15.2
 MYSQL_VERSION=5.7
```

Comment:
Probably should change this to use v8 instead of v5.7 since 5.7 is EOL

Reply:
agreed, i did run it against 8, but there's a bug/issue with connecting to 8 with the adapter. 
https://github.com/crystal-lang/crystal-mysql/issues/62

I suggest we handle moving to 8 for testing in a separate issue, as it will take me a bit to work around it.